src/App.vue

```diff
@@ -449,12 +450,12 @@ export default class App extends Vue {
       firstName: isStaff ? '' : this.getUserFirstName,
       lastName: isStaff ? '' : this.getUserLastName,
       mailingAddress: {
-        addressCity: isStaff ? '' : this.getOrgInfo?.mailingAddress.city,
-        addressCountry: isStaff ? '' : this.getOrgInfo?.mailingAddress.country,
-        addressRegion: isStaff ? '' : this.getOrgInfo?.mailingAddress.region,
-        postalCode: isStaff ? '' : this.getOrgInfo?.mailingAddress.postalCode,
-        streetAddress: isStaff ? '' : this.getOrgInfo?.mailingAddress.street,
-        streetAddressAdditional: isStaff ? '' : this.getOrgInfo?.mailingAddress.streetAdditional
+        addressCity: isStaff ? '' : this.getOrgInfo?.mailingAddress?.city,
+        addressCountry: isStaff ? '' : this.getOrgInfo?.mailingAddress?.country,
+        addressRegion: isStaff ? '' : this.getOrgInfo?.mailingAddress?.region,
+        postalCode: isStaff ? '' : this.getOrgInfo?.mailingAddress?.postalCode,
+        streetAddress: isStaff ? '' : this.getOrgInfo?.mailingAddress?.street,
+        streetAddressAdditional: isStaff ? '' : this.getOrgInfo?.mailingAddress?.streetAdditional
```

Comment:
Are you sure this extra conditional chaining is needed -- did you see an example of an org without a mailing address object?

Reply:
was changed to pass the unit test, will try it without and see if it can be fixed

Reply:
Reverted, thanks.